src/Chartjs/assets/dist/controller.js

```diff
@@ -34,8 +34,14 @@ class default_1 extends Controller {
     }
     viewValueChanged() {
         if (this.chart) {
-            this.chart.data = this.viewValue.data;
-            this.chart.options = this.viewValue.options;
+            const data = this.viewValue.data;
+            let options = {};
+            if (!Array.isArray(this.viewValue.options)) {
```

Comment:
I'm not sure to get this line ?

Reply:
if `$chart->setOptions([...])` provided then the value of `this.viewValue` will be `{ key: "value" }` if no options provided the value will be `[]` only

Reply:
made modification to the code

if you try to remove this line https://github.com/symfony/ux/pull/1605/files#diff-4bf03a7251075e059ffaeb81d3465ab2277cc6438c3e5c729b3ce4d22cbf332dR39-R41 the test will be failed with error that I put on the description